specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/client.options.sasl.scram-256.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "test",
+    "bindings":
+    {
+        "app0":
+        {
+            "type" : "kafka",
+            "kind": "client",
+            "options":
+            {
+                "sasl":
+                {
+                    "mechanism": "scram-sha-1",
```

Comment:
Configuration is called `scram-256` but mechanism is `scram-sha-1`, please make them consistent, and create a new one only if we need both.